rhoas-cli/README.adoc

```diff
@@ -151,30 +127,43 @@ $ rhoas --version
 
 .Procedure
 
-. Review the link:https://github.com/redhat-developer/app-services-cli/blob/main/scripts/install.sh[`install.sh` installation script^].
+. Navigate to the https://developers.redhat.com/products/red-hat-openshift-streams-for-apache-kafka/overview[Red Hat OpenShift Streams for Apache Kafka] page on the Red Hat Developer Portal.
+
+. Click https://developers.redhat.com/products/red-hat-openshift-streams-for-apache-kafka/download[*Install/Specifics*] and navigate to the bottom of the page to the *All Downloads* section.
```

Comment:
```suggestion
. Click https://developers.redhat.com/products/red-hat-openshift-streams-for-apache-kafka/download[*Install/specifics*] and scroll to the bottom of the page to the *All Downloads* section.
```

Reply:
Same as above, this is the exact link title from the devportal page.